control/tests/matlab_test.py

```diff
@@ -400,9 +400,8 @@ def testModred(self):
         modred(self.siso_ss3, [1], 'matchdc')
         modred(self.siso_ss3, [1], 'truncate')
 
-    @unittest.skipIf(not slycot_check(), "slycot not installed")
     def testPlace(self):
-        place(self.siso_ss1.A, self.siso_ss1.B, [-2, -2])
```

Comment:
I think we should leave this test in place, but calling `place_varga` (assuming `slycot` is installed).  Also, can we add a check for the right type of error if we call `place` with repeated poles?

Reply:
I have made this change. I have also added two place-holder tests (or whatever they are called) into matlab_test.py for place and acker.

 I added an assert_raises(ValueError....) test for place() to statefbk_test.py.